platform/accessibility/common-issues/pdf/table-missing-headers.md

```diff
@@ -0,0 +1,76 @@
+# [PDF]: Table is missing headers
+
+## Table of contents
+
+  - [Descrption of the issue](#description-of-the-issue)
+  - [Definition of done](#definition-of-done)
+  - [Steps to fix the issue](#steps-to-fix-the-issue)
+    - [Define table headers in Microsoft Word](#define-table-headers-in-microsoft-word)
+    - [Define table headers in Acrobat Pro](#define-table-headers-in-acrobat-pro)
+  - [Test for compliance](#test-for-compliance)
+    - [Microsoft Word accessibility panel](#microsoft-word-accessibility-panel)
+    - [Acrobat Pro accessibility tools](#acrobat-pro-accessibility-tools)
+  - [Learn more](#learn-more)
+
+## Description of the issue
+
+- Synopsis: Make tables accessible in PDF documents by using table elements. The applicable standard from Section508.22.h dictates to "associate data and header cells for multi-dimensional data tables."
+- Severity: Critical
+- Deque University help link: https://dequeuniversity.com/rules/comply-pdf/1.0/table-miss-header
+- WCAG Success Criteria: [1.3.1 Info and Relationships](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)
+
+## Definition of done
+
+- There SHOULD be **0 table header violations** in the Word [Check Accessibility panel](https://support.microsoft.com/en-us/office/improve-accessibility-with-the-accessibility-checker-a16f6de0-2f39-4a2b-8bd8-5ad801426c7f)
+- There MUST be  in the Acrobat [Accessibility Checker](https://www.adobe.com/accessibility/products/acrobat/using-acrobat-pro-accessibility-checker.html)
```

Comment:
Seems like this line is missing something...?

Reply:
Yep, the bolded portion didn't get carried over. Fixing now.